src/analysis/retail/priest/holy/modules/talents/BottomRow/ResonantWords.tsx

```diff
@@ -55,10 +57,17 @@ class ResonantWords extends Analyzer {
   }
 
   onHeal(event: HealEvent) {
-    if (this.selectedCombatant.hasBuff(SPELLS.RESONANT_WORDS_BUFF.id)) {
+    if (this.selectedCombatant.hasBuff(SPELLS.RESONANT_WORDS_TALENT_BUFF.id)) {
       this.usedResonantWords += 1;
-      const eventBonusAmount = event.amount - event.amount / (1 + this.healingMultiplier);
-      this.bonusHealing += eventBonusAmount;
+
+      const overhealing = event.overheal != null ? event.overheal : 0;
+      const absorbed = event.absorbed != null ? event.absorbed : 0;
+      const totalHealing = event.amount + overhealing + absorbed;
+
+      const totalhealingFromTalent = totalHealing - totalHealing / (1 + this.healingMultiplier);
+      this.overhealingDoneFromTalent +=
+        overhealing <= totalhealingFromTalent ? overhealing : totalhealingFromTalent;
+      this.healingDoneFromTalent += Math.max(totalhealingFromTalent - overhealing, 0);
```

Comment:
We have a helper for this calculation, see `calculateEffectiveHealing`

Reply:
Added new function for calculating overhealing and used both